src/components/FirstTimeUserFlow/PropertyPrefForm.js

```diff
@@ -0,0 +1,115 @@
+import * as React from 'react';
+import { useState } from 'react';
+import Typography from '@mui/material/Typography';
+import {
+	Grid,
+	FormControlLabel,
+	FormControl,
+	FormLabel,
+	Slider,
+	FormGroup,
+	Checkbox,
+} from '@mui/material/';
+
+import { amenitiesConst } from '@/constants/constants';
+
+const amenityOptions = amenitiesConst;
+
+export default function PropertyPrefForm() {
+	const priceMarks = [
+		{
+			value: 500,
+			label: '$500',
+		},
+		{
+			value: 1500,
+			label: '$1500',
+		},
+		{
+			value: 2500,
+			label: '$2500',
+		},
+		{
+			value: 3500,
+			label: '$3500',
+		},
+	];
+	const distanceMarks = [
+		{
+			value: 5,
+			label: '5',
+		},
+		{
+			value: 15,
+			label: '15',
+		},
+		{
+			value: 25,
+			label: '25',
+		},
+		{
+			value: 35,
+			label: '35',
+		},
+		{
+			value: 45,
+			label: '45',
+		},
+	];
```

Comment:
this is totally fine and can be merged, but there's gotta be a better way of generating these arrays something like `[500, 1500, 2500, 3500].map(x => { value: x, label: "$" + x.toString() }`? but not a blocker, it's fine to merge